tests/acceptance/features/coreApiSharePublicLink3/allowGroupToCreatePublicLinks.feature

```diff
@@ -49,18 +49,18 @@ Feature: public share sharers groups setting
     And parameter "public_share_sharers_groups_allowlist_enabled" of app "files_sharing" has been set to "yes"
     And parameter "public_share_sharers_groups_allowlist" of app "files_sharing" has been set to '["grp1"]'
     When user "Alice" updates the last public link share using the sharing API with
-      | expireDate | +3 days |
+      | expireDate | 2040-01-01T23:59:59+0100 |
```

Comment:
Was this necessary?
Can we only assert the date (`2040-01-01`) but not the time in order to cope with the differences? That way we may use incremental days `+3`.

Reply:
I did this for more transparency. If we use this aproach In the test, we can see what date format we use in the body of the request and what date format we get in the response. Unfortunately, we have a difference in this (I created a bug), and I have to postpone PR until it's clearer how it's supposed to work

Reply:
we get a different date format in the response, so I:
- created error #5442
- check the date format (2040-01-01)